anatomy/tracks/practice-exercises.md

```diff
@@ -28,4 +28,334 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
+
+- Test suite: verify a solution's correctness.
+- Stub implementation: provide a starting point for students.
+- Example implementation: provide an example implementation that passes all the tests.
+- Additional files: ensure that the tests can run.
+
+### Example
+
+<pre>
+exercises
+└── practice
+    └── isogram
+        ├── .docs
+        |   ├── introduction.md
+        |   ├── instructions.md
+        |   └── hints.md
+        ├── .meta
+        |   ├── config.json        
+        |   ├── design.md
+        |   └── Example.cs (example implementation)
+        ├── Isogram.cs (stub implementation)
+        └── IsogramTests.cs (tests)
+</pre>
+
+---
+
+### File: `.docs/introduction.md`
+
+**Purpose:** Introduce the setting and background for the exercise to the student.
+
+**Presence:** Required if the exercise implements a Problem Specifications Exercise with an `introduction.md` file
+
+If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `introduction.md` file. configlet has functionality to automatically (sync the contents)[./configlet/generating-documents.md#documentpracticeexercisesintroductionmdfile] of this file.
+
+If the exercise is _not_ based on a Problem Specifications Exercise, consider the following:
+
+We place high value on making Exercism's content safe for everyone and so often err on the side of caution in deciding whether stories are appropriate or not. While we are careful about what we merge, we appreciate that it's hard to be aware of what may be seen as problematic, so we'll always assume you're acting in good faith and do our best to catch any issues in review in a non-confrontational way. If you'd like to check a story with us, please mention @exercism/leadership and we'll look at it together. Here are some guiding points:
+
+- Try to make sure the story is welcoming and can be understood by everyone. If the story contains in-jokes or regional slang, try to think of alternative phrases.
+- Try to write examples that are inclusive to everyone. For example, consider using names from other cultures and mixed genders.
+- Ask yourself whether you know anyone personally who would take offense by the story. If that's the case, consider changing it to avoid it.
+
+#### Example
+
+```markdown
+Bob is a lackadaisical teenager. In conversation, his responses are very limited.
+```
+
+---
+
+### File: `.docs/introduction.append.md`
+
+**Purpose:** Additional introduction text to append after the existing introduction.
+
+**Presence:** Optional
+
+In some (rare) cases, you might want to expand on the exercise's `introduction.md` file, for example when the exercise has implemented tests that are not covered by the existing instructions.
+
+```markdown
+Bob has been having a tough time lately though, so you want to help Bob.
+```
```

Comment:
Can we think of an example here that is more realistic based on being language-specific? Maybe:

_(The suggestion formatting is off here due to a GH bug)_

```suggestion
A track that doesn't not want to support unicode for Bob, might add the following:

```markdown
As part of his teenage rebellion, Bob has decided to only communicate using ASCII
```
```